apps/personal-library/db/BookModel.js

```diff
@@ -1,14 +1,14 @@
-const Mongoose = require("mongoose");
-
-const BookSchema = new Mongoose.Schema({
-  title: String,
-  comments: Array,
-  commentcount: Number,
-});
-
-const BookModel = Mongoose.model("book", BookSchema);
-
-module.exports = {
-  BookSchema,
-  BookModel,
-};
+const Mongoose = require("mongoose");
```

Comment:
Not sure why GitHub took exception to this one.  CRLF, maybe?  Either way, it's unchanged if you ignore spaces.